mtags/src/main/scala/scala/meta/internal/mtags/SemanticdbClasspath.scala

```diff
@@ -2,22 +2,25 @@ package scala.meta.internal.mtags
 
 import java.nio.charset.Charset
 import java.nio.charset.StandardCharsets
+import java.nio.file.Path
 
 import scala.meta.AbsolutePath
-import scala.meta.Classpath
 import scala.meta.internal.mtags
 import scala.meta.internal.mtags.ScalametaCommonEnrichments._
 import scala.meta.internal.mtags.Semanticdbs.FoundSemanticDbPath
+import scala.meta.io.Classpath
 import scala.meta.io.RelativePath
 
 final case class SemanticdbClasspath(
     sourceroot: AbsolutePath,
     classpath: Classpath = Classpath(Nil),
+    semanticdbTargets: List[Path] = Nil,
```

Comment:
Turns out this is only needed if running sbt BSP since the semanticdb files will end up in a separate path by default in that case.